src/main/webapp/app/view/my-projects/my-projects.component.html

```diff
@@ -41,16 +41,47 @@ <h2 id="page-heading">
          [infiniteScrollDisabled]="page >= links['last']" [infiniteScrollDistance]="0">
         <div *ngFor="let projectRow of projects" class="row">
             <div *ngFor="let project of projectRow;trackBy: trackId" class="col-md">
-                <a [routerLink]="['/project', project.id, 'view']">
-                    <div *ngIf="project.id" class="project-card">
+                <div *ngIf="project.id">
+                    <div *appHasAnyRole="{ projectId: project.id, roles: adminRoles}; else readOnlyProjectCard"
+                         class="project-card">
                         <h2>{{project.name}}</h2>
                         <br/>
                         <div class="project-card-description">
                             {{project.description ? project.description
                             : 'eventPlannerApp.project.mine.noDescription' | translate}}
                         </div>
+                        <br/>
+                        <button id="jh-screenplay-{{project.id}}"
+                                [routerLink]="['project',project.id,'screenplay']"
+                                class="btn btn-negative">
+                            <fa-icon [icon]="faBook"></fa-icon>
+                            <span jhiTranslate="eventPlannerApp.project.home.screenplayLabel">
+                            Screenplay
+                        </span>
+                        </button>
+                        <button id="jh-settings-{{project.id}}"
+                                [routerLink]="['project',project.id]"
+                                class="btn btn-negative">
+                            <fa-icon [icon]="faCog"></fa-icon>
+                            <span jhiTranslate="eventPlannerApp.project.home.settingsLabel">
+                            Settings
+                        </span>
+                        </button>
                     </div>
-                </a>
+
+                    <ng-template #readOnlyProjectCard>
+                        <a [routerLink]="['/project', project.id, 'screenplay']">
+                            <div class="project-card">
+                                <h2>{{project.name}}</h2>
+                                <br/>
+                                <div class="project-card-description">
+                                    {{project.description ? project.description
+                                    : 'eventPlannerApp.project.mine.noDescription' | translate}}
+                                </div>
+                            </div>
+                        </a>
+                    </ng-template>
+                </div>
```

Comment:
Ich würde in einem nächsten Schritt die Reihenfolge der Project Cards noch anpassen. Zurzeit steht das Projekt mit der kleinsten project.id am Anfang. Nehmen wir an jemand hat 20 Projekte, davon sind bereits 15 abgeschlossen. Dann möchte er nicht die 5 Projekte an denen er Arbeitet ganz unten haben. Würde es wohl nach Datum sortieren. Das nächste Projekt welches ansteht zu erst. Und Projekte die schon abgeschlossen sind am Schluss

Reply:
das habe ich mir auch schon überlegt. machen wir einen folgetask? man könnte das mit so einer art "archiv-funktion" auch kombinieren.

Reply:
oder man könnten natürlich user-preferences mit einer reihenfolge speichern. dann drag n' drop :) aber das wäre richtig over engineering..

Reply:
Ja würde es in einem neuen Task machen. Würde die Zeit lieber in anderes stecken als das drag n' drop

Reply:
erfasst in #55. merge diesen PR, damit wir den Grundstein gelegt haben.